docs/guides/specifying-upstream-dependencies.md

```diff
@@ -0,0 +1,247 @@
+---
+description: Run tasks in a desired order
+tags:
+  - tasks
+  - dependencies
+  - wait_for
+search:
+  boost: 2
+---
+
+# Specifying Upstream Dependencies
+
+Results from a task can be provided to other tasks (or subflows) as upstream dependencies. Prefect uses upstream dependencies in two ways:
+
+1. To populate dependency arrows in the flow run graph
+2. To determine execution order for [concurrently submitted](/concepts/task-runners) units of work that depend on each other
+
+!!! note "Tasks vs. other functions"
+    **Only results from tasks** inform Prefect's ability to determine dependencies. Return values from functions without task decorators, including subflows, do not carry the same information about their origin as task results.
+
+When using non-sequential task runners such as the [`ConcurrentTaskRunner`](/api-ref/prefect/task-runners/#prefect.task_runners.ConcurrentTaskRunner) or [`DaskTaskRunner`](https://prefecthq.github.io/prefect-dask/), the order of execution for submitted tasks is not guaranteed unless their dependencies are specified.
+
+For example, compare how tasks submitted to the `ConcurrentTaskRunner` behave with and without upstream dependencies:
```

Comment:
```suggestion
For example, compare how tasks submitted to the `ConcurrentTaskRunner` behave with and without upstream dependencies by clicking on the tabs below.
```

Reply:
I didn't notice there was a tab for a bit. I think I like the tabs, but could work fine w/o them (just inline, too). Your call.